BE/OwnershipAndControl/Executives.rdf

```diff
@@ -256,171 +307,93 @@
+		<fibo-fnd-utl-av:explanatoryNote>For a public, for profit organization, its members (directors) are elected normally by the subscribers (stockholders) of the firm (generally at an annual general meeting) to govern the firm and look after the subscribers&apos; interests. In the case of a not-for-profit organization, the board ensures the organization is acting in line with its mission. The board has the ultimate decision-making authority and, in general, is empowered to (1) set the company&apos;s policy, objectives, and overall direction, (2) adopt bylaws, (3) name members of the advisory, executive, finance, and other committees, (4) hire, monitor, evaluate, and fire the managing director and senior executives, (5) determine and pay the dividend, and (6) issue additional shares. Though all its members might not be engaged in the company&apos;s day-to-day operations, the entire board is held liable (under the doctrine of collective responsibility) for the consequences of the firm&apos;s policies, actions, and failures to act. Members of the board usually include senior-most executives (called inside directors or executive directors) as well as experts or respected persons chosen from the wider community (called outside directors or non-executive directors).</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-be-oac-exec;Bylaw">
+	<owl:Class rdf:about="&fibo-be-oac-exec;Bylaws">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-law-cor;Law"/>
-		<rdfs:label>bylaw</rdfs:label>
-		<skos:definition>Corporate bylaws are typically called Articles of Association in the United Kingdom, or Articles of Incorporation in the United States. In government usage, the term bylaws is used to describe laws, orders, regulations, rules, etc., made by a ministry, local authority (a municipality, for example), or public corporation, in accordance with the powers conferred by or delegated under a statue (called the parent act).</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/bylaw.html</fibo-fnd-utl-av:adaptedFrom>
+		<rdfs:label>bylaws</rdfs:label>
+		<skos:definition>written rules for conduct of a corporation, association, partnership or any organization</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Corporate bylaws are typically called Articles of Association in the United Kingdom, or Articles of Incorporation in the United States. In government usage, the term bylaws is used to describe laws, orders, regulations, rules, etc., made by a ministry, local authority (a municipality, for example), or public corporation, in accordance with the powers conferred by or delegated under a statue (called the parent act).</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
I think Articles and Bylaws are different: indeed we have a Property (mentioned earlier) to link them (though I don't think the property is useful). See https://www.upcounsel.com/bylaws-vs-articles-of-incorporation#:~:text=The%20Articles%20of%20Incorporation%20include,procedures%20for%20managing%20the%20corporation.

Maybe have as synonym "membership agreement"?

Reply:
I'll kill the note and add the synonym.

Reply:
I agree that they are distinct from the articles of incorporation, at least in the US